### DISCUSSION TITLE
Scavenger Hunt Mission : Show us the view

### DISCUSSION THREAD

**Discussion by radhikasethi2011** [education/hackconviii#9] [General]
Share a picture of your “view” at the conference. Laptop on the couch? Maybe the weather's nice and you're hanging out outside? Let's see what HackCon looks like in your slice on the universe.

My desk fo the evening! 

![IMG_20200822_184539](https://user-images.githubusercontent.com/44310177/90956883-00f64680-e4a8-11ea-9f46-f9729b2ebb80.jpg)

**Comment by mohdsaabith** [comment]
![WhatsApp Image 2020-08-22 at 21 10 23](https://user-images.githubusercontent.com/44156514/90960631-a0bfce80-e4c0-11ea-82f2-ba9e16b3d267.jpeg)

**Comment by BazilNizam** [comment]
![photo_2020-08-23_16-26-45](https://user-images.githubusercontent.com/22983667/90976775-740dc480-e55d-11ea-9472-2bab1d2619df.jpg)
